### ISSUE TITLE
evaluate Hugo as replacement for Webby

### ISSUE THREAD

**Issue by phrawzty** [devopsdays/devopsdays-web#4]
http://gohugo.io/

Seems to tick a number of boxes (though I've never used it myself):
* Modern; actively maintained.
* Static binary (instead of Ruby dependency / version hell).
* Some sort of template mechanism.
* Supports Markdown out of the box.

What's more, @mattstratton has [operational experience](http://discuss.gohugo.io/t/arrested-devops-is-live-with-our-new-hugo-based-site/1871) with it, so that'd certainly be helpful.

That said, if there's something that's better, I'm not married to Hugo - worth looking at though!

**Comment by m1keil** [comment]
Taking a look at https://www.staticgen.com/ it seems to me that Hugo is the best choice for us today.

If I understand correctly, the main issue we are trying to solve here is the fact that the site is being generated by old unmaintained project which is very hard to install due to old requirements on any platform.

Since Hugo is written in Go, it makes the installation and usage extremely easy. Just download binary and run on supported platform (Linux, OS X, Windows).

The only disadvantage I know of with using Hugo is the fact that Go doesn't have dynamic loading, which limits the possibilities for plugins as opposed to more dynamic languages such as Ruby or Python. However I don't think it's a real show stopper here since DOD website is very simple.

So I think we should just proceed with Hugo. :smile:

**Comment by mattstratton** [comment]
Obviously my vote is for Hugo, since I've worked with it. I also did a bunch of different static site evaluation when I moved ADO recently, and Hugo is where I landed for basically similar reasons (I needed a simple way where content could be accomplished via simple PR's, and I have content contributors who need an easy way to do it).

The Hugo community is very active and it's a very actively developed project, so I have a high level of comfort using it as the going-forward.
